docs/tutorials/benchmarking/README.md

```diff
@@ -0,0 +1,17 @@
+# Benchmarking in Syne Tune
+
+
+This tutorial provides an overview of tooling which facilitates benchmarking of
+HPO algorithms in Syne Tune. The same tooling can also be used to rapidly
+create launcher scripts for any HPO experiment you may want to run, allowing you
```

Comment:
nitpick: remove "you may want to run" 
long sentences are hard to grasp.

Reply:
OK